config/routes.rb

```diff
@@ -1,4 +1,15 @@
 Rails.application.routes.draw do
+  get 'authors/show'
+  get 'authors/index'
```

Comment:
Delete these two lines since you are doing that on line 5

Reply:
Done